extensions/Cake.ExtractLicenses.yml

```diff
@@ -0,0 +1,24 @@
+Type: Addin
+Name: Cake.ExtractLicenses
+NuGet: Cake.ExtractLicenses
+Assemblies:
+- /**/Cake.ExtractLicenses.dll
+Repository: https://github.com/wgnf/liz.git
+ProjectUrl: https://github.com/wgnf/liz/
+Author: wgnf
```

Comment:
Can this name be changed to "Martin Wagenführ"?

Reply:
@wgnf Yes, this will be updated automatically in a future PR generated by our tool (after we merge this one)